rails/app/controllers/api/v1/permission_forms_controller.rb

```diff
@@ -2,37 +2,47 @@ class API::V1::PermissionFormsController < API::APIController
 
   # GET /api/v1/permission_forms/index
   def index
-    @permission_forms = policy_scope(Portal::PermissionForm)
-    render :json => @permission_forms
+    authorize Portal::PermissionForm, :permission_forms_v2_index?
+    permission_forms = managment_policy_scope(Portal::PermissionForm)
```

Comment:
typo: `management_policy_scope`

Reply:
Thanks, it already got fixed in https://github.com/concord-consortium/rigse/pull/1343 ;)